main.go

```diff
@@ -165,6 +175,17 @@ type configInput struct {
 	secretsDir         string
 	metadataLabels     map[string]string
 	deploymentInfo     map[string]string
+	configScope        string
+}
+
+func validate(in configInput) error {
+	re := regexp.MustCompile(`^[a-zA-Z][a-zA-Z0-9-]{0,63}$`)
+	if in.configScope != "" && !re.MatchString(in.configScope) {
+		return fmt.Errorf("config-scope may only contain letters, numbers, and '-'. " +
+			"It must begin with a letter and must exceed 64 characters in length")
```

Comment:
must _not_ exceed

Reply:
Indeed. What an odd restriction that would be.